packages/faustwp-cli/src/healthCheck/validateFaustEnvVars.ts

```diff
@@ -28,4 +28,28 @@ export const validateFaustEnvVars = () => {
       'Please make sure your production Faust app runs with a WordPress instance on https!',
     );
   }
+  if (getWpSecret()) {
+    // send secret key
+    const apiUrl = `${getWpUrl()}/wp-json/faustwp/v1/validate_secret_key`;
```

Comment:
Related to https://github.com/wpengine/faustjs/issues/1735

```suggestion
    const apiUrl = `${getWpUrl()}/?rest_route=/faustwp/v1/validate_secret_key`;
```